orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -77,15 +77,45 @@ class OperationsController {
 
   @RequestMapping(value = "/orchestrate", method = RequestMethod.POST)
   Map<String, Object> orchestrate(@RequestBody Map pipeline, HttpServletResponse response) {
-    parsePipelineTrigger(executionRepository, buildService, pipeline)
-    Trigger trigger = objectMapper.convertValue(pipeline.trigger, Trigger)
+    if (pipeline.plan) {
+      parseAndValidatePipeline(pipeline)
+      log.info('Not starting pipeline (plan: true): {}', value("pipelineId", pipeline.id))
+      return pipeline
```

Comment:
I realize we have `Map<String, Object>` here but the long expected result from this endpoint is either:
(a) an exception
or
(b) ["ref": "/.../{id}"]

It seems a little strange to echo the pipeline json back

Reply:
Echoing back the pipeline when `pipeline.plan` is true was the behavior prior to this change; if we think it makes sense to change, I can do so, but didn't want to change that behavior as part of this change.  I agree that it seems a bit strange.

Reply:
Oops ... yes I just checked on that behavior and was coming back in here to remove this comment.

'tis a bit confusing but we can leave as is for now.